src/main/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSink.java

```diff
@@ -274,36 +294,61 @@ public void configure(Context context) {
-        
+        String attrPersistenceStr = context.getString("attr_persistence");
+
+
         if (fileFormatConfigured) {
-            fileFormat = FileFormat.valueOf(context.getString("file_format").replaceAll("-", "").toUpperCase());
-            LOGGER.debug("[" + this.getName() + "] Reading configuration (file_format=" + fileFormat + ")");
+            String fileFormatStr = context.getString("file_format");
+            
+            try {
+                fileFormat = FileFormat.valueOf(fileFormatStr.replaceAll("-", "").toUpperCase());
+                LOGGER.debug("[" + this.getName() + "] Reading configuration (file_format="
+                    + fileFormatStr + ")");
+            } catch (Exception e) {
+                invalidConfiguration = true;
+                LOGGER.debug("[" + this.getName() + "] Invalid configuration (file_format="
+                    + fileFormatStr + ") -- Must be 'json-row', 'json-column', 'csv-row' or 'csv-column'");
+            } // catch
+            
         } else if (rowAttrPersistenceConfigured) {
-            boolean rowAttrPersistence = context.getString("attr_persistence").equals("row");
-            LOGGER.debug("[" + this.getName() + "] Reading configuration (attr_persistence="
-                + (rowAttrPersistence ? "row" : "column") + ") -- DEPRECATED, converting to file_format="
-                + (rowAttrPersistence ? "json-row" : "json-column"));
-            fileFormat = (rowAttrPersistence ? FileFormat.JSONROW : FileFormat.JSONCOLUMN);
+            if (attrPersistenceStr.equals("row") || attrPersistenceStr.equals("column")) {
+                boolean rowAttrPersistence = attrPersistenceStr.equals("row");
+                LOGGER.debug("[" + this.getName() + "] Reading configuration (attr_persistence="
+           		+ attrPersistenceStr + ") -- DEPRECATED, converting to file_format="
+           		+ (rowAttrPersistence ? "json-row" : "json-column"));
+            } else {
+                invalidConfiguration = true;
+                LOGGER.debug("[" + this.getName() + "] Invalid configuration (attr_persistence="
+                    + attrPersistenceStr + ") -- Must be 'row' or 'column'");
+            }  // if else
+            
         } else {
             fileFormat = FileFormat.JSONROW;
             LOGGER.debug("[" + this.getName() + "] Defaulting to file_format=json-row");
         } // if else if
-        
+
         // Hive configuration
-        String enableHiveStr = context.getString("hive", "true");
-        enableHive = enableHiveStr.equals("true");
-        LOGGER.debug("[" + this.getName() + "] Reading configuration (hive=" + (enableHive ? "true" : "false")
-                + ")");
+        String enableHiveStr = context.getString("hive");
         
+        if (enableHiveStr.equals("true") || enableHiveStr.equals("false")) {
+            enableHive = Boolean.valueOf(enableHiveStr);
```

Comment:
Unnecssary, since enableHiveStr can be directy used, as done in the `else` branch.